portal/package.json

```diff
@@ -16,6 +16,7 @@
     "start": "node server/index.js",
     "dev": "webpack --config webpack.config.dev.js --watch",
     "test": "jest --coverage --verbose --config=jest.config.js",
+    "test-dev": "jest --verbose --config=jest.config.js",
```

Comment:
Is there a reason for this service?

Reply:
we have it in the others.  Useful when running tests locally as coverage takes up too much room